data/json/recipes/recipe_others.json

```diff
@@ -1170,6 +1170,20 @@
     "tools": [ [ [ "welder", 50 ], [ "welder_crude", 75 ], [ "toolset", 75 ], [ "oxy_torch", 10 ] ] ],
     "components": [ [ [ "wire", 6 ], [ "basket", 1 ] ], [ [ "scrap", 8 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "bike_basket",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": 5000,
```

Comment:
I guess 8 minutes is rather low to create a wire basket. You need to cut wires, bend them, and weld. Maybe 30 or 60 minutes will be better.